### DISCUSSION TITLE
Phase 2 plans

### DISCUSSION THREAD

**Discussion by wtrocki** [aerogear/charmil#75] [Ideas]
My recomendations for the next weeks

- review each others PR - focusing on your work is ok but reviews are equally important - let's work on reviews along with mentors and discuss
- Collect more and more info into issues as we go. 
- Discuss/Discuss every idea that you have. New feature? == New issue or discussion :slight_smile:
- Create small PR's regularly that are easy and fast to merge

I like we getting quite good roadmap so if we can start using projects and assign issues that will be also an additional benefit for you.

Best teams work with some daily communication.
I have created voice and video channel on discord called - charmil-daily-chat
Feel free to use that between yourselves and mentors

**Comment by ankithans** [comment]
Week 2 Summary/Report
https://ankithans.github.io/gsoc21/weekly-reports/week2.html

Overall Tracker for all weeks
https://ankithans.github.io/gsoc21/

**Reply by wtrocki** [reply]
mind blown
